### PR TITLE
Add test cases for JSON resource marshaling

### PR DESCRIPTION
**Pull request by sagar23sj**
Resources Covered :

- ServiceHook
- CodeOfConduct
- markdownRequest

Helps with issue: #55

### CODE REVIEW

File: github/misc_test.go
Comment (gmlewis) [inline github/misc_test.go:352]:
```suggestion
func TestMarkdownRequest_Marshal(t *testing.T) {
```

File: github/misc_test.go
Comment (gmlewis) [inline github/misc_test.go:370]:
```suggestion
func TestCodeOfConduct_Marshal(t *testing.T) {
```

File: github/misc_test.go
Comment (gmlewis) [inline github/misc_test.go:390]:
```suggestion
func TestServiceHook_Marshal(t *testing.T) {
```

**Review by gmlewis — CHANGES_REQUESTED** [review]

**Review by gmlewis — APPROVED** [review]
Thank you, @sagar23sj !
LGTM.

Awaiting second LGTM before merging.

**Review by Parker77 — APPROVED** [review]
LGTM.

**Pull request merged**
